e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -132,170 +133,174 @@ internal static class TestHelper {
         /// </summary>
         /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
         /// <returns>Instance of SshClient, that need to be disposed</returns>
-        private static SshClient CreateSshClientAndConnect(IIoTPlatformTestContext context) {
-            context.OutputHelper?.WriteLine("Load private key from environment variable");
-            var privateKeyStream = GetPrivateSshKey(context.SshConfig);
-            var privateKeyFile = new PrivateKeyFile(privateKeyStream);
+        private static ScpClient CreateScpClientAndConnect(IIoTPlatformTestContext context) {
+            var privateKeyFile = GetPrivateSshKey(context);
 
-            context.OutputHelper?.WriteLine("Create SSH Client");
-            var client = new SshClient(
+            context.OutputHelper?.WriteLine("Create SCP Client");
+            var client = new ScpClient(
                 context.SshConfig.Host,
                 context.SshConfig.Username,
                 privateKeyFile);
 
-            context.OutputHelper?.WriteLine("open ssh connection to host {0} with username {1}",
+            context.OutputHelper?.WriteLine("open scp connection to host {0} with username {1}",
                 context.SshConfig.Host,
                 context.SshConfig.Username);
             client.Connect();
-            context.OutputHelper?.WriteLine("ssh connection successful established");
+            context.OutputHelper?.WriteLine("scp connection successful established");
 
             return client;
         }
 
         /// <summary>
-        /// Delete published_nodes.json file into the OPC Publisher edge module
+        /// Gets the private SSH key from the configuration to connect to the Edge VM
         /// </summary>
-        /// <param name="destinationFilePath">Destination file path</param>
         /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
-        public static void DeletePublishedNodesFile(string destinationFilePath, IIoTPlatformTestContext context) {
+        /// <returns></returns>
+        private static PrivateKeyFile GetPrivateSshKey(IIoTPlatformTestContext context)
+        {
+            context.OutputHelper?.WriteLine("Load private key from environment variable");
+
+            var buffer = Encoding.Default.GetBytes(context.SshConfig.PrivateKey);
+            var privateKeyStream = new MemoryStream(buffer);
+
+            var privateKeyFile = new PrivateKeyFile(privateKeyStream);
+            return privateKeyFile;
+        }
+
+        /// <summary>
+        /// Delete a file on the Edge VM
+        /// </summary>
+        /// <param name="fileName">Filename of the file to delete</param>
+        /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
+        public static Task DeleteFileOnEdgeVM(string fileName, IIoTPlatformTestContext context) {
```

Comment:
Why do we need the method to return `Task` if there are no asynchronous calls in it ?